pom.xml

```diff
@@ -111,6 +116,13 @@
           <version>3.0.0</version>
           <scope>provided</scope>
         </dependency>
+        <!-- Fix test error due to fail to transform class AbstractProject when using javassist-3.18.1-GA -->
+        <dependency>
+            <groupId>org.javassist</groupId>
+            <artifactId>javassist</artifactId>
+            <version>3.19.0-GA</version>
+            <scope>test</scope>
```

Comment:
Ensure that there is no non-test dependency on this lib. Otherwise explicit dependency with the `test` scope will override these deps and make the library to be excluded from hpi

Reply:
[matrix-project-dependency-tree.txt](https://github.com/jenkinsci/matrix-project-plugin/files/1382880/matrix-project-dependency-tree.txt)
Attached the dependency tree when `mvn dependency:tree` is executed on master. As you can see there is only one reference to javassist and it is test scope.